_rules/SC1-1-1-image-has-name.md

```diff
@@ -23,9 +23,9 @@ The rule applies to HTML `img` elements or any HTML element with the [semantic r
 
 ### Expectation
 
-Each target element has an [accessible name](#accessible-name) that is [non-empty](#non-empty) or is marked as [decorative](#decorative).
+Each target element has an [accessible name](#accessible-name) that is not only [whitespace](#whitespace), or is marked as [decorative](#decorative).
```

Comment:
@kasperisager and @WilcoFiers, can you both agree to this change? 

> does not only consist of [unicode separator characters]()

--> 

> is not only [whitespace]() (+ definition of "whitespace")

If yes, I will go ahead and make this change in all rules that uses this wording.